### ISSUE TITLE
[a11y]: missing attributes and placeholder text 

### ISSUE THREAD

**Issue by CBID2** [codedex-io/html-101#52]
### Description
In the [19_contact_us.html file](https://github.com/codedex-io/html-101/blob/main/3-forms/19_contact_us.html), I found the following accessibility issues:

- [ ] the `<label>` elements are missing `for` attributes. This ensures that the information that the user types are correctly associated with the input.
- [ ] the input fields are missing placeholder text and `id` attributes, causing users to have difficultly with understanding what to type.
